memcached-spring-boot-autoconfigure/build.gradle

```diff
@@ -10,10 +10,12 @@ artifacts {
 
 dependencies {
     implementation('org.springframework.boot:spring-boot-autoconfigure')
+    implementation('org.hibernate.validator:hibernate-validator')
```

Comment:
Could you please revert the property validation as it was, and remove dependency on `hibernate-validator`. The aim is of the library is to have minimal dependencies, except the dependencies required by Spring. That way the users of the library would not need to download additional dependency with their app.